public/components/common/search/sql_search.tsx

```diff
@@ -241,20 +241,32 @@ export const DirectSearch = (props: any) => {
 
   useEffect(() => {
     // cancel direct query
-    if (pollingResult && (pollingResult.status === 'SUCCESS' || pollingResult.datarows)) {
-      // stop polling
-      stopPolling();
-      setIsQueryRunning(false);
-      dispatch(
-        updateSearchMetaData({
-          tabId,
-          data: {
-            isPolling: false,
-          },
-        })
-      );
-      // update page with data
-      dispatchOnGettingHis(pollingResult, '');
+    if (pollingResult) {
+      if (pollingResult.status === 'SUCCESS' || pollingResult.datarows) {
```

Comment:
replace 'SUCCESS' and other status with enum?

Reply:
replaced